specutils/analysis/template_comparison.py

```diff
@@ -138,7 +147,14 @@ def template_match(observed_spectrum, spectral_templates,
         The index of the spectrum with the smallest chi2 in spectral templates.
     """
     if hasattr(spectral_templates, 'flux') and len(spectral_templates.flux.shape) == 1:
-        normalized_spectral_template, chi2 = _template_match(
+
+        # Account for redshift if provided
+        if min_redshift and max_redshift and delta_redshift:
```

Comment:
```suggestion
        if all(x is not None for x in (min_redshift, max_redshift, delta_redshift)):
```

In this case you're checking identity of the parameters, not the truthiness, so you should use the `is` operator.